pythonwhat/State.py

```diff
@@ -55,62 +52,12 @@ def __init__(self, **kwargs):
         self.fun_usage = None
         self.manual_sigs = None
```

Comment:
@machow it does exist, but unlike the converters it's passed inside `test_function_v2`. Might make sense to pull it out similar to `converters` (which I find more elegant)